docs/UserGuide.md

```diff
@@ -1196,3 +1215,6 @@ Below is a summary of all the commands available in the WerkIt! application.
 | Search For Plan          | <br /> Format: `search /plan <plan keyword>` <br/>Example: `search /plan arms` <br /> <br />                                                                   |
 | Search For ALL           | <br /> Format: `search /all <keyword>` <br/>Example: `search /all 10` <br /> <br />                                                                            |
 | View Help                | <br /> Format: `help`<br/>Example: `help`  <br /> <br />                                                                                                       |
+
+
+[1]:https://www.microsoft.com/en-us/p/windows-terminal/9n0dx20hk701
```

Comment:
oo why is this link here :O?

Reply:
I learned that you can use 'variable-like' referencing for links. Since this URL is used twice, I thought might as well create a single 'variable' with the link and use the variable in the other parts (in 'Quick Start Guide' and FAQ).

Reply:
thank you for the clarification!